doc/release-notes/master.txt

```diff
@@ -16,6 +16,22 @@ Vital
  * Added a foreach_pixel helper function to apply an operation at every
    pixel of a const image.
 
+ * Added the data_serializer base algorithm to support serilization
+   of Vital types for distributed processing and other uses.
+
+ * Applied a new approach to adding metadata to plugins. The plugin
+   name and description are now specified in the plugin
+   header. Added new plugin registrar class framework to simplify the
+   registration process.
+
+Vital Tools
+
+ * Converted tools to use applet plugin approach.
+
+ * Converting stand-alone tools to reside under the main "kwiver" top
```

Comment:
Not sure this belongs in this PR, be meh... however, shouldn't it be "converted", not "converting"?